core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -299,12 +287,32 @@ def detect_frontend_to_install(bundler_context)
         MSG
     end
 
-    def support_solidus_frontend_extraction(bundler_context)
+    def support_solidus_frontend_extraction
       say_status "break down", "solidus"
 
       SupportSolidusFrontendExtraction.
         new(bundler_context: bundler_context).
         call
     end
+
+    def install_plugin(plugin_name:, plugin_generator_name:)
+      if bundler_context.dependencies[plugin_name]
+        say_status :skipping, "#{plugin_name} is already in the gemfile"
+        return
+      end
+
+      gem plugin_name
+      BundlerContext.bundle_cleanly { run "bundle install" }
```

Comment:
As I wrote in the commit message:

>   * Installing the plugin sequentially would call `bundle install` twice,
      but this is okay since InstallGenerator in master also adds gems to the
      bundle one at a time.

Reply:
nit: since it's now using `AbbBase` as the superclass I think the whole line can be replaced with `run_bundle`

Reply:
I guess then we could remove the whole `BundlerContext.bundle_cleanly` method, right?

Reply:
I'll try changing that call to `run_bundle` and removing `bundle_cleanly` :+1:

Reply:
I was able to remove `bundle_cleanly` but I had to call `bundle_command` with `#send` in InstallFrontend. Please see https://github.com/solidusio/solidus/pull/4748#discussion_r1036860556. Thanks!